typescript/src/liquidityBootstrapping/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './data';
 export * from './liquidityBootstrapping';
+export * from './liquidityBootstrappingMath';
```

Comment:
I'm wondering where you'll use this. Usually we rely on balancer-maths to do the math as a "black box" and simply work with its results.

Reply:
As part of the pool implementation in the SOR, I need to implement getPoolState. This function returns the weights, which I can either fetch from the api (cached) or calculate it on the fly (as done in maths). I am not sure which I should use, but since weights are calculated in the maths package, I figured I might as well calculate them in the SOR instead of using api results.

Reply:
If you're calculating weights within balancer-maths, poolState should pass inputs (i.e. `startWeights`, `endWeights`, `startTime` and `endTime`) so weights can be calculated, instead of SOR calculating them and passing them on the poolState, right?

Reply:
Yeah, you are right. I didn't fully catch the "mapping" purpose of `getPoolState`. I'm closing this pr as not needed.